helm/alfresco-content-services/values.yaml

```diff
@@ -529,6 +529,11 @@ alfresco-digital-workspace:
     # APP_CONFIG_MICROSOFT_ONLINE_CLIENTID:
     # APP_CONFIG_MICROSOFT_ONLINE_AUTHORITY:
     # APP_CONFIG_MICROSOFT_ONLINE_REDIRECT:
+  extraEnv: |-
+    {{- if .Values.global.ai.enabled }}
+    - name: APP_CONFIG_PLUGIN_AI_SERVICE
+      value: '{{ .Values.global.ai.enabled }}'
+    {{- end }}
```

Comment:
@aymanthefirst fails how ? 

I think the green builds essentially run "helm install" without specifying "global.ai.enabled". So I'm assuming an empty "extraVars" block is ok. 

Alternatively, we might need to move the if/end around the "extraVars" (until we introduce other variables) or even shift to within the "env" block instead, eg. something like:

`APP_CONFIG_PLUGIN_AI_SERVICE: {{- if .Values.global.ai.enabled }}'{{ .Values.global.ai.enabled }}'{{else}}'false'{{-end }}`

Reply:
I can't say how it fails as I never did find a way to provide more useful logs than this:
`Administrator@Ayman MINGW64 ~/Documents/AlfrescoRepos/acs-deployment (feature/ATS-884-add-ai-to-pipeline)
$ git commit -am"testing"
Helm Docs................................................................Failed
- hook id: helm-docs
- exit code: 1

Please install helm-docs to run the pre-commit hook! https://github.com/norwoodj/helm-docs#installation

Travis Lint..........................................(no files to check)Skipped
Helm Dependency Update...................................................Failed
- hook id: helm-deps
- exit code: 123

helm dep up ./helm/alfresco-content-services
Error: no 'repository' field specified for dependency: "activemq"
helm dep up ./helm/alfresco-content-services/charts/activemq
No requirements found in C:\Users\Administrator\Documents\AlfrescoRepos\acs-deployment\helm\alfresco-content-services\charts\activemq/charts.
helm dep up ./helm/alfresco-content-services/charts/alfresco-search
Error: no 'repository' field specified for dependency: "alfresco-insight-zeppelin"
helm dep up ./helm/alfresco-content-services/charts/alfresco-search/charts/alfresco-insight-zeppelin
No requirements found in C:\Users\Administrator\Documents\AlfrescoRepos\acs-deployment\helm\alfresco-content-services\charts\alfresco-search\charts\alfresco-insight-zeppelin/charts.
helm dep up ./helm/alfresco-content-services/charts/alfresco-sync-service
No requirements found in C:\Users\Administrator\Documents\AlfrescoRepos\acs-deployment\helm\alfresco-content-services\charts\alfresco-sync-service/charts.

Helm Lint................................................................Failed
- hook id: helm-lint
- exit code: 123

helm lint ./helm/alfresco-content-services
==> Linting ./helm/alfresco-content-services
[ERROR] templates/: render error in "alfresco-content-services/templates/deployment-repository.yaml": template: alfresco-content-services/templates/deployment-repository.yaml:24:28: executing "alfresco-content-services/templates/deployment

-repository.yaml" at <include (print $.Template.BasePath "/config-repository.yaml") .>: error calling include: template: alfresco-content-services/templates/config-repository.yaml:127:110: executing "alfresco-content-services/templates/con

fig-repository.yaml" at <.Values.postgresql.service.port>: nil pointer evaluating interface {}.port

Error: 1 chart(s) linted, 1 chart(s) failed
helm lint ./helm/alfresco-content-services/charts/activemq
==> Linting ./helm/alfresco-content-services/charts/activemq
Lint OK

1 chart(s) linted, no failures
helm lint ./helm/alfresco-content-services/charts/alfresco-search
==> Linting ./helm/alfresco-content-services/charts/alfresco-search
Lint OK

1 chart(s) linted, no failures
helm lint ./helm/alfresco-content-services/charts/alfresco-search/charts/alfresco-insight-zeppelin
==> Linting ./helm/alfresco-content-services/charts/alfresco-search/charts/alfresco-insight-zeppelin
Lint OK

1 chart(s) linted, no failures
helm lint ./helm/alfresco-content-services/charts/alfresco-sync-service
==> Linting ./helm/alfresco-content-services/charts/alfresco-sync-service
Lint OK

1 chart(s) linted, no failures

Checkov - Helm check.....................................................Failed
- hook id: checkov-helm
- exit code: 1

Error: found in requirements.yaml, but missing in charts/ directory: postgresql, postgresql, common`

Reply:
Reviewed with @aymanthefirst. The failed commit was due to missing readme update (pre-commit helm-docs).

In theory, we could remove if/end block but it would mean APP_CONFIG_PLUGIN_AI_SERVICE is always set (false or true). However, for now, I suggest we leave as-is.